virttest/libvirt_xml/devices/filesystem.py

```diff
@@ -29,6 +29,8 @@ def __init__(self, type_name='mount', virsh_instance=base.base.virsh):
         accessors.XMLElementDict('alias', self,
                                  parent_xpath='/',
                                  tag_name='alias')
+        accessors.XMLAttribute('boot', self, parent_xpath='/',
```

Comment:
We could consider XMLElementDict if there are new attributes to 'boot' element in the future. 
And we could keep the current implement if the chance of adding new attribute is really small.

Reply:
I think we can keep it in current status because it is unlikely that there will be a new attribute to 'boot' element.  If there are indeed changes later, we can update them together.

Reply:
Of course.